models/linkages/R/read.restart.LINKAGES.R

```diff
@@ -0,0 +1,26 @@
+read.restart.LINKAGES <- function(outdir,run.id,time,IC,prior,spinup){
+  
+  forecast = IC
+  nens = nrow(IC)
+  ens = list()
+  
+  for(i in 1:nens){
```

Comment:
Another design question: had we intended the generic read.restart.MODEL to be passed a vector of run.id's for a whole ensemble or to be run one-at-a-time for each ensemble member?  Your code does the former but the latter is how write.config is designed. To me it seems more intuitive to put the loop over ensemble members in the main code, rather than requiring each model to implement this looping and construction of X, but we may have discussed otherwise earlier.